Andreas changes put back in order to adust to the 3.0 phis-ws and the 3.0 user doc releases

pierreetiennealary opened the pull request.

pierreetiennealary committed:
POST images CSV tempalte changed in order to match the 2.7 WS release

pierreetiennealary committed:
POST images Python script changed in order to match both the 2.7 WS release and the 2.7 user doc release

Andreas-Garcia commented:
Ok. I just changed the target release number  which will be the 3.0 (and not the 2.7).

It seems that you have tagged the user doc with the number "2.7". You may also consider to change it to the "3.0"  version

pierreetiennealary commented:
> Ok. I just changed the target release number which will be the 3.0 (and not the 2.7).
> 
> It seems that you have tagged the user doc with the number "2.7". You may also consider to change it to the "3.0" version

I just changed the name of the user-doc PR to "newest changes put back in order to match 3.0 release of both the webapp and the web services" in order to match the name that we will give to the next releases
